Migrate monitoring to Slinky repo

dylanschultzie opened the pull request:
This will need massaged a bit once @ericjohncarlson gets his dashboard in. 

High level things to look at:
- paths
- oracle.json downloading (should I just reference the oracle.json under `config/dydx/oracle.json`?

dylanschultzie committed:
Migrate monitoring to Slinky

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

Commit:
Merge branch 'main' into main

Zygimantass commented on docker-compose.yml:
```diff
@@ -0,0 +1,151 @@
+    entrypoint: [
+      "slinky", 
+      "--oracle-config-path", "/etc/slinky/oracle.json", 
+      # "--market-config-path", "/oracle/market.json", 
```
are these comments supposed to be here @nivasan1 ?

g3ndo9kar9 replied:
No, IMO we should template these parameters so I can change the environment based on arguments?

dylanschultzie committed:
Add dydx dashboard

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

g3ndo9kar9 commented on docker-compose.yml:
```diff
@@ -0,0 +1,151 @@
+      org.label-schema.group: "monitoring"
+
+  slinky:
+    image: ghcr.io/skip-mev/slinky-sidecar:v0.4.1
```
latest tag

g3ndo9kar9 commented on monitoring/.env.sample:
```diff
@@ -0,0 +1,11 @@
+POLL_SECONDS=15
+
+# Slinky Dashboard
+DASHBOARD_URL=https://raw.githubusercontent.com/RhinoStake/slinky_monitoring_dashboard/main/Slinky%20Oracle%20Dashboard.json
```
Are these all safe to share?

g3ndo9kar9 replied:
can we point the dashboard URL to the slinky repo?

dylanschultzie replied:
> Are these all safe to share?

Yes, the admin password/login is simply `admin/admin`, which requires these properties.

g3ndo9kar9 commented on monitoring/.gitignore:
```diff
@@ -0,0 +1,5 @@
+.env
```
can we move these to the main repo's git ignore

davidterpay committed:
Merge branch 'main' into main

davidterpay committed:
Merge branch 'main' into main

davidterpay committed:
Merge branch 'main' into main

dylanschultzie committed:
Fix grafana locations

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

dylanschultzie committed:
Remove slinky command comments

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

dylanschultzie committed:
Clean up readme

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

dylanschultzie committed:
Update .env locations

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

dylanschultzie committed:
Clean up deployment

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

dylanschultzie commented:
@nivasan1 addressed your comments, thanks for that. 

how would you like me to handle deployment instructions? right now the readme is under `monitoring/readme.md`, but does it make more sense to have a top level `deploy.md`?

dylanschultzie committed:
Merge branch 'skip-mev:main' into main

dylanschultzie committed:
Remove references to the downloader container

dylanschultzie committed:
Clean up readme

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

dylanschultzie committed:
Update default metrics to localhost

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

dylanschultzie committed:
Re-add node ip bits

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

dylanschultzie committed:
Update latest slinky container

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

dylanschultzie committed:
Merge branch 'main' into main

dylanschultzie committed:
Rewrite readme for if only using the monitoring tooling

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

dylanschultzie committed:
Update dydx dashboard

Signed-off-by: Dylan Schultz <9121234+dylanschultzie@users.noreply.github.com>

davidterpay committed:
Merge branch 'main' into main

Eric-Warehime committed:
Merge branch 'main' into main

technicallyty commented:
closing as stale, will revisit if/when we allot dev cycles for monitoring.
